docker/testing/ubuntu2004_j11.docker

```diff
@@ -26,10 +26,15 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
     apt-get install -y --no-install-recommends software-properties-common apt-utils vim && \
     apt-get install -y lsof
 
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    add-apt-repository ppa:git-core/ppa && \
+    apt-get update && \
+    apt-get install -y git
```

Comment:
what version of git do you need?
what version is ubuntu:20.04 providing? 

we want a clear comment about this being temporary and when it can be removed.

Reply:
ubuntu provides 2.25.1
we need at least 2.30.0
the command currently installs 2.41.0

tbh I don't know why wouldn't we ever use the newest one